Author: Holger Jaekel <holger.jaekel@gmx.de>
Summary: ignore external directory as we want to use provided libraries
Upstream: No
----

--- a/makefile
+++ b/makefile
@@ -22,7 +22,7 @@
 #
 # Sub-directories that need to be built
 #
-subdirs	= external vpflib ogdi contrib
+subdirs	= vpflib ogdi contrib
 
 #
 # Default target to build everything in all sub-directories


--- a/ogdi/makefile
+++ b/ogdi/makefile
@@ -15,7 +15,7 @@
 #
 # Sub-directories that need to be built
 #
-subdirs	= include c-api glutil driver gltpd examples
+subdirs	= include c-api glutil driver gltpd
 
 #
 # Default target to build everything in all sub-directories
